frontends/web/src/locales/en/app.json

```diff
@@ -781,6 +781,48 @@
         "title": "Which service should I use?"
       }
     },
+    "manageBB02": {
+      "backups": {
+        "link": {
+          "text": "Learn more about managing backups",
+          "url": "https://guides.shiftcrypto.ch/bitbox02/basics/managing-backups/"
+        },
+        "text": "On the 'Manage backups' page you can verify that your microSD card backup is valid and optionally create additional backups.",
+        "title": "What can I do on the 'Manage backups' page?"
+      },
+      "mnemonic": {
+        "link": {
+          "text": "Learn more about displaying your mnemonic seed",
+          "url": "https://guides.shiftcrypto.ch/bitbox02/advanced/view-seed/"
+        },
+        "text": "After clicking, you will first have to confirm on your BitBox02 that you want to display your mnemonic seed by entering your device password.\nOnce confirmed, your BitBox02 will display your mnemonic seed so that you can write it down.\nWhen done, you then have to confirm each seed word to ensure that you have properly written them down.",
```

Comment:
Please rename 'mnemonic' to 'recovery words', see
https://github.com/digitalbitbox/bitbox-wallet-app/pull/963/files